woocommerce/woocommerce
# Add woocommerce_order_received_verify_known_shoppers filter to order confirmation template

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

WooCommerce 8.4.0 introduced `woocommerce_order_received_verify_known_shoppers` filter for Order received page. This PR implements that filter for the Order Confirmation template.





Closes #45136 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Using the Code Snippets extensions, add the following snippet: `add_filter('woocommerce_order_received_verify_known_shoppers', '__return_false' );`
2. Using a block theme and Checkout block, place an order.
3. Get the confirmation page url.
4. Open a new incognito page, paste the url.
5. You should be able to see the order.
6. Deactivate the snippet.
7. Refresh the page on the incognito page, you should NOT be able to see the order.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
